Rakefile

```diff
@@ -25,7 +25,13 @@ namespace :style do
 
   desc 'Run Chef style checks'
   FoodCritic::Rake::LintTask.new(:chef) do |t|
-    t.options = { fail_tags: ['correctness'] }
+    t.options = {
+      fail_tags: ['correctness'],
+      tags: [
+        '~FC048', # Disables: Prefer shell_out helper method to shelling out with ruby
```

Comment:
Why do we disable this one? I assume `FC121` is disabled because we want to support Chef <14 but I am not sure about `FC048`.

Reply:
I was just being lazy, I've fixed the code and re-enabled the check.

Reply:
Looks like it is not available on Chef <14 either 😞